src/ActionList/List.tsx

```diff
@@ -124,19 +125,27 @@ export function List(props: ListProps): JSX.Element {
    */
   const renderGroup = (
     groupProps: GroupProps | (Partial<GroupProps> & {renderItem?: typeof Item; renderGroup?: typeof Group})
-  ) => ((('renderGroup' in groupProps && groupProps.renderGroup) ?? props.renderGroup) || Group).call(null, groupProps)
+  ) => {
+    const GroupComponent = (('renderGroup' in groupProps && groupProps.renderGroup) ?? props.renderGroup) || Group
+    return <GroupComponent {...groupProps} key={groupProps.groupId} />
```

Comment:
🔥

Reply:
I like the symmetry you preserved between `renderGroup` and `renderItem`. Nice work @VanAnderson!